vogon/urls.py

```diff
@@ -79,71 +78,71 @@
+    re_path(rf'^{settings.APP_ROOT}repository/(?P<repository_id>[0-9]+)/collections/(?P<group_id>[0-9]+)/$', views.repository_views.repository_collection, name='repository_collection'),
+    re_path(rf'^{settings.APP_ROOT}repository/(?P<repository_id>[0-9]+)/collection/(?P<group_id>[0-9]+)/group-collection/(?P<group_collection_id>[a-zA-Z0-9]+)/texts/$', views.repository_views.repository_collection_texts, name='repository_collections_text_list'),
+    re_path(rf'^{settings.APP_ROOT}repository/(?P<repository_id>[0-9]+)/group/(?P<group_id>[a-zA-Z0-9]+)/text/(?P<text_key>[a-zA-Z0-9]+)/$', views.repository_views.repository_text_import, name='repository_text_import'),
+    re_path(rf'^{settings.APP_ROOT}repository/(?P<repository_id>[0-9]+)/text/(?P<text_id>[0-9]+)/content/(?P<content_id>[0-9]+)/$', views.repository_views.repository_text_content, name='repository_text_content'),
+    re_path(rf'^{settings.APP_ROOT}repository/(?P<repository_id>[0-9]+)/$', views.repository_views.repository_details, name='repository_details'),
+    re_path(rf'^{settings.APP_ROOT}repository/(?P<repository_id>[0-9]+)/text/(?P<text_id>[0-9]+)/project/(?P<project_id>[0-9]+)/$', views.repository_views.repository_text_add_to_project, name='repository_text_add_to_project'),
 
-    re_path(r'^repository/$', views.repository_views.repository_list, name='repository_list'),
+    re_path(rf'^{settings.APP_ROOT}repository/$', views.repository_views.repository_list, name='repository_list'),
 
-    re_path(r'^text/(?P<text_id>[0-9]+)/public/$', views.text_views.text_public, name='text_public'),
+    re_path(rf'^{settings.APP_ROOT}text/(?P<text_id>[0-9]+)/public/$', views.text_views.text_public, name='text_public'),
 
-    #re_path(r'^annotate/image/(?P<text_id>[0-9]+)/$', views.annotation_views.annotate_image, name='annotate_image'),
+    #re_path(rf'^{settings.APP_ROOT}annotate/image/(?P<text_id>[0-9]+)/$', views.annotation_views.annotate_image, name='annotate_image'),
 
-    re_path(r'^sandbox/(?P<text_id>[0-9]+)/$', conceptViews.sandbox, name='sandbox'),
+    re_path(rf'^{settings.APP_ROOT}sandbox/(?P<text_id>[0-9]+)/$', conceptViews.sandbox, name='sandbox'),
 
-    path('login/citesphere/', externalAccountViews.citesphere_login, name='citesphere_login'),
-    path('oauth/callback/citesphere/', externalAccountViews.citesphere_callback, name='citesphere_callback'),
-    path('citesphere/refresh/<int:repository_id>/', externalAccountViews.citesphere_refresh_token, name='citesphere_refresh_token'),
+    path(f'{settings.APP_ROOT}login/citesphere/', externalAccountViews.citesphere_login, name='citesphere_login'),
+    path(f'{settings.APP_ROOT}oauth/callback/citesphere/', externalAccountViews.citesphere_callback, name='citesphere_callback'),
+    path(f'{settings.APP_ROOT}citesphere/refresh/<int:repository_id>/', externalAccountViews.citesphere_refresh_token, name='citesphere_refresh_token'),
```

Comment:
let's follow the approach from dst and prefix all urls in one go (not one in each url)
https://github.com/diging/dst/blob/develop/code/dst/urls.py